README-ruRU.md

```diff
@@ -3792,9 +3792,9 @@
 # Сотрудничество
 
 Ничто, описанное в этом руководстве, не высечено в камне. И я очень хотел бы
-сотрудничать со всеми, кто интересуется стилистикой оформления кода на Руби,
+сотрудничать со всеми, кто интересуется стилистикой оформления кода на Ruby,
 чтобы мы смогли вместе создать ресурс, который был бы полезен для всего
-сообщества программистов на Руби.
```

Comment:
We do not have any prescriptions at this point. Transctiption and transliteration are the straightforward way, but not always the best and habitual. Russian tends to have it's own rules like "микроскоп", "Генрих Гейне". Java was always an allude to the island and the coffee, which is called "Ява" in Russian. And to keep this connection it could be a good choice. I repeat, there is no strict rule, only the tradition and private preferences (mine is "Ява").